aeron-archive/src/main/java/io/aeron/archive/ReplicationSession.java

```diff
@@ -526,7 +541,7 @@ private int extend()
                 recordingSubscription.asyncAddDestination(replayDestination);
             }
 
-            state(State.REPLAY_TOKEN);
+            state(State.REPLAY_TOKEN, "");
```

Comment:
Should be "replay destination added" probably including `replayDestination` value.